tests/cephfs/cephfs_utilsV1.py

```diff
@@ -1234,6 +1236,96 @@ def remove_nfs_export(
                     )
         return cmd_out, cmd_rc
 
+    def create_osd_pool(
```

Comment:
This change is also part of another PR, so it may add conflict during merge? please check on this

Reply:
Which ever goes in first based on that I will rebase the next PR Suma